packages/backend/src/telemetry/service.ts

```diff
@@ -173,14 +173,13 @@ export class TelemetryServiceImpl implements TelemetryService {
         return
       }
 
-      const obfuscatedAmount = preservePrivacy
-        ? privacy.applyPrivacy(Number(converted))
-        : Number(converted)
-      this.incrementCounter(name, obfuscatedAmount, attributes)
+      const finalAmount = preservePrivacy
+        ? privacy.applyPrivacy(Number(converted.amount))
```

Comment:
ah man... `Number` takes anything.

Reply:
Definitely feels like it should take in some proper types instead...